rules/manager_test.go

```diff
@@ -112,7 +112,7 @@ func TestAlertingRule(t *testing.T) {
 		if actual[0] == "" {
 			actual = []string{}
 		}
-		testutil.Equals(t, expected, actual)
+		testutil.Assert(t, len(expected) == len(actual), "%d. Number of samples in expected and actual output don't match (%d vs. %d)", i, len(expected), len(actual))
```

Comment:
Is switching from comparing the exact result to just checking the length the right thing to do here?

Reply:
Yes, I converted it wrong the first time.
https://github.com/prometheus/prometheus/blob/dev-2.0/rules/manager_test.go#L124